### PR TITLE
fix flag for windows

### PR DESCRIPTION
**Pull request by MoLow**
after a few hours trying to figure out why I cant run this on a windows machine,  I found these changes will make it work.

 BTW, the tests cant even start on windows, so I couldent run them locally either :(

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -96,6 +96,12 @@ Here are the instructions for running Firefox from the command line (MacOS + Fir
 $ /Applications/Firefox.app/Contents/MacOS/firefox-bin --start-debugger-server 6080 -P development
 ```
 
+**note** - In windows you should replace --start-debugger-server with -start-debugger-server:
```
Comment (clarkbw) [inline CONTRIBUTING.md]: Can you put a header up for each OS instead of this line?

Something like this:

**MacOS**
```
/App...
```

**Windows**
```
C:\Program Files...
```

File: docs/remotely-debuggable-browsers.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ These are the flags necessary to start the remote debug server and use an altern
 * debug server `--start-debugger-server 6080`
 * separate profile `-P development` (not required)
  * **Note**: if you are prompted with the profile manager you will need to create a profile named `development`
+ * **Note**: in windows replace `--start-debugger-server 6080` with `-start-debugger-server 6080`
```
Comment (clarkbw) [inline docs/remotely-debuggable-browsers.md]: We have a Windows section below which gives the right options.  Maybe we need a TOC to make it clearer what options we have available.

**Comment by clarkbw** [comment]
@MoLow Thanks!  

It looks like #721 just added instructions for Windows testing

File: docs/remotely-debuggable-browsers.md (hunk 2 above)
Comment (clarkbw) [inline docs/remotely-debuggable-browsers.md]: Here's a TOC that might make it easier to find everything in this doc.  I don't think the links are all working, but its close if you want to include it.


#### Table Of Contents

[Firefox](#firefox)
  * [MacOS](#macos)
    * [Firefox Release](#firefoxrelease)
    * [Firefox Beta](#firefox-beta)
    * [Firefox Developer Edition](#firefox-developer-edition)
    * [Firefox Nightly](#firefox-nightly)
  * [Windows](#windows)
    * [Firefox (all versions)](#firefox-all-versions)
    * [Firefox (64 bit)](#firefox-64-bit)
  * [Android](#android)

[Chrome](#chrome)
  * [MacOS](#macos)
    * [Chrome (release)](#chromerelease)
    * [Chrome Canary](#chrome-canary)
  * [Windows](#windows)
    * [Chrome (all versions)](#chromeallversions)
    * [Chrome (64 bit)](#chrome-64-bit)

Comment (jlongster) [inline docs/remotely-debuggable-browsers.md]: A table of contents sounds good. Not sure why you need the 4 different versions under MacOS though?

Comment (jlongster) [inline docs/remotely-debuggable-browsers.md]: @clarkbw Do you want to take on adding the table of contents in a separate issue/pr?

Comment (clarkbw) [inline docs/remotely-debuggable-browsers.md]: Yeah, I can open that up separately

**Comment by clarkbw** [comment]
This looks good to go then

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -88,15 +88,17 @@ $ npm run firefox
 * Restart your development server `ctrl+c` in the Terminal and run `npm start` again
 * Reload `localhost:8000` (you should now see a Firefox tabs section)
 
-**Command line option**
+*Command line option**
```
Comment (clarkbw) [inline CONTRIBUTING.md]: Just need to revert this change

**Comment by clarkbw** [comment]
Thanks @MoLow 🎆 
The tests failed on the mochitest run for unrelated reasons, other than the driver change this is just changes to the docs.

**Comment by jasonLaster** [comment]
Thanks @MoLow. I appreciate your resilience. Please feel free to snag one of our [up for grabs](https://github.com/devtools-html/debugger.html/issues?q=is%3Aissue+is%3Aopen+label%3A%22up+for+grabs%22) issues!
